src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,36 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.ArrayList;
```

Comment:
It is good practice to have `import` statements in alphabetically order. Intellij auto-import feature can help in this.